src/plone/testing/zca.py

```diff
@@ -31,11 +31,11 @@ def _hookRegistry(reg):
     globalregistry.base = reg
     globalregistry.globalSiteManager = reg
 
-    # Set the default global site manager for new threads when zope.site's
+    # Set the default global site manager for new threads when zope.component
     # hooks are in place
 
     try:
-        from zope.site.hooks import SiteInfo
+        from zope.component.hooks import SiteInfo
```

Comment:
zope.component cannot fail. But the next one, five.localsitemanager, might simply not be available...